virtcontainers/api_test.go

```diff
@@ -94,6 +93,7 @@ func newTestSandboxConfigNoop() SandboxConfig {
 	return sandboxConfig
 }
 
+<<<<<<< HEAD
```

Comment:
@gabibeyer This needs to be deleted I guess :)

Reply:
🤦‍♀️ oops